bindgen/codegen/mod.rs

```diff
@@ -4133,6 +4134,20 @@ impl CodeGenerator for Function {
                 quote! { #[link(wasm_import_module = #name)] }
             });
 
+        if ctx.options().generate_extern_functions.is_second_run() {
```

Comment:
That is, why can't we just add this + the link_name in the first run and call it a day?

Reply:
you were right!